### DISCUSSION TITLE
How can I set yazi as default file/folder picker?

### DISCUSSION THREAD

**Discussion by sachinsenal0x64** [sxyazi/yazi#281] [1. Q&A] [answered]
OS: Arch Linux 64 bit
WM: Qtile

**Comment by sxyazi** [comment, marked as answer]
- https://github.com/Eric-Song-Nop/fm-nvim
- https://github.com/DreamMaoMao/yazi.nvim
- https://github.com/GermainZ/xdg-desktop-portal-termfilechooser

Not sure if those are what you want

**Comment by sachinsenal0x64** [comment]
Hey again yazi.nvim one working fine so i need xdg-desktop-portal-termfilechooser but its not working i tried everything.

**Comment by sxyazi** [comment]
Ah sorry, I haven't actually used `xdg-desktop-portal-termfilechooser`. In fact, I'm not even a Linux user; I just came across this link on the Yazi Discord.

It seems like it hasn't been maintained for quite some time, and unfortunately, I can't do anything about that. :)

**Comment by sachinsenal0x64** [comment]
its ok no problem :)

**Comment by exquo** [comment]
Here's an attempt at a more comprehensive [README](https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser/blob/8259ceeacf92e009e0763832944027b6763bbdf2/README.md) for `termfilechooser`, and here is a [yazi wrapper script](https://github.com/boydaihungst/xdg-desktop-portal-termfilechooser/blob/3ca52f7b0135eed375b05d5fed84d589f0620af5/contrib/yazi-wrapper.sh) for it. Hope this helps!
